storage/wrapper.go

```diff
@@ -0,0 +1,168 @@
+package storage
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+)
+
+type ctxKey string
+
+const (
+	DBCounterCtxKey ctxKey = "db_counter"
```

Comment:
This used to be `db_stats`, i wouldn't change it or we'll have to remember when querying old data

Also, do we need db stats at all in OpenFGA?

Reply:
I think the msg key used to be `db_stats`. I think a whole log line dedicated to logging db_stats is a bit overkill and adds too much noise. In any case, I don't think we actually ever queried on it did we? We do have a dashboard widget with it, but, again, I don't think it adds much value.